src/components/addbook.css

```diff
@@ -4,7 +4,8 @@
 
 .addbook-form {
   width: 86rem;
-  margin: 2.313rem 6.25rem 0;
+  margin: 2.313rem 6.25rem 2rem;
+  padding-bottom: 100px;
 }
```

Comment:
[REQUIRED] Kindly implement the requirements in the image below:

| Template |
|----------|
|![image](https://user-images.githubusercontent.com/62938735/191816441-a644e0c5-3ad8-4940-baf9-6502e361462b.png)|